packages/discovery-provider/plugins/notifications/src/processNotifications/mappers/comment.ts

```diff
@@ -93,6 +96,13 @@ export class Comment extends BaseNotification<CommentNotificationRow> {
 
     const title = 'New Comment'
     const body = `${commenterUserName} commented on your ${entityType.toLowerCase()} ${entityName}`
+
+    // Get commenter's profile picture URL for rich notification (150x150 size)
+    const commenterUser = users[this.commenterUserId]
+    const imageUrl = commenterUser.profile_picture_sizes
```

Comment:
instead of using profile_picture_sizes, you can just use the `profile_picture` object returned
https://api.audius.co/v1/users/handle/rayjacobson

and skip the `formatImageUrl` call. 

not blocking this PR though, that should work